client/components/Photo.jsx

```diff
@@ -1,4 +1,17 @@
-import React from 'react';
+import React from "react";
+import ReactDOM from "react-dom";
 
+class Photo extends React.Component {
```

Comment:
I think this can be functional stateless in its current form. constructor() { super() } on lines 5-7 don't do any work right now, so I think this class can be transformed into a function that just returns the JSX on lines 10-13.